ethers-core/src/types/trace/erigon.rs

```diff
@@ -0,0 +1,34 @@
+use ethabi::ethereum_types::U256;
+use serde::{Deserialize, Serialize};
+
+use crate::types::{transaction::eip2718::TypedTransaction, BlockNumber};
+
+#[derive(Serialize, Debug, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct EthCallManyBlockOverride {
+    pub block_number: U256,
+}
+
+#[derive(Serialize, Debug, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct EthCallManyOptions<'a> {
+    pub block_number: &'a BlockNumber,
+}
+
+#[derive(Serialize, Debug, Clone)]
+pub struct EthCallManyBalanceDiff {
+    pub balance: U256,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct EthCallManyResponse {
+    pub value: Option<String>,
```

Comment:
this is called output I believe

and should be `Bytes`

Reply:
Updated to Option<Bytes>